plugins/prospector.cpp

```diff
@@ -168,9 +168,13 @@ static void printMatdata(color_ostream &con, const matdata &data, bool only_z =
         con << std::setw(9) << int(data.count);
 
     if(data.lower_z != data.upper_z)
-        con <<"   Z:" << std::setw(4) << data.lower_z << ".." <<  data.upper_z << std::endl;
+        con <<"   Z:" << std::setw(4) << data.lower_z << " .." << std::setw(4) << data.upper_z
+            <<"   Elev:" << std::setw(4) << (data.lower_z - 100) << " .." << std::setw(4) << (data.upper_z - 100)
```

Comment:
I was going to ask about the regional offset, but I see this was already discussed here: https://discord.com/channels/793331351645323264/1017471248567124049/1088569309149798441